resources/patches/ruby-stomp/connection_loss_reconnect_fix.patch

```diff
@@ -1,125 +0,0 @@
-From ed5da4d1d857c427bf995e50c9baa25befaa3175 Mon Sep 17 00:00:00 2001
```

Comment:
This patch is present at https://github.com/stompgem/stomp/commit/dea38701d93dd15dac59b3b5dbabb3df2df9dde0.